single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DirectionalFeederInfo.java

```diff
@@ -19,9 +19,9 @@
  */
 public class DirectionalFeederInfo extends AbstractFeederInfo {
 
-    private final LabelProvider.LabelDirection arrowDirection;
+    private LabelProvider.LabelDirection arrowDirection;
```

Comment:
If possible we should keep those final.

Reply:
Yes, I agree. For that, I must perform the reverse within the label provider.